app/views/follows/_follow_button.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="js-follow">
   <span class="followable-content">
-    <% if follow.followable.followed_by?(current_user) %>
+    <% if follow.followable.class.name=="Community" ? current_user && current_user.follows.present? : follow.followable.followed_by?(current_user) %>
```

Comment:
Please, amend this line to make it look like this: `class.name == "Community"`

Reply:
done